server/modules/record/manager/_recordManager/recordUpdateManager.js

```diff
@@ -206,7 +202,7 @@ const _updateNodeAndValidateRecordUniqueness = async (
     const { record: recordUpdated1, nodes: nodesUpdated } = await nodesUpdateFn(user, survey, record, node, t)
     let recordUpdated = recordUpdated1
 
-    const { record: recordUpdated2, updatedNodesAndDependents } = await _onNodesUpdate(
+    const { record: recordUpdated2, nodes: updatedNodesAndDependents } = await _onNodesUpdate(
```

Comment:
recordUpdated2 this two and the 1 make me feel weird

Reply:
eheheh, I didn't want to change too much the code... I will change it in the next PR! the result of _onNodesUpdate is actually a RecordUpdateResult object, so I can replace it with that (next PR)